src/js/dom/dom.js

```diff
@@ -0,0 +1,31 @@
+const searchModalOpenButton = document.querySelector('#search-modal-open-button');
+const modalClose = document.querySelector('.modal-close');
+const modal = document.querySelector('.modal');
+const youtubeSearchForm = document.querySelector('.youtube-search-form');
+const youtubeSearchInput = document.querySelector('.youtube-search-input');
+const modalVideoWrapper = document.querySelector('.modal .video-wrapper');
+const mainVideoWrapper = document.querySelector('main > .video-wrapper');
+const modalInner = document.querySelector('.modal-inner');
+const recentSearchList = document.querySelector('.recent-search-list');
+const playListLength = document.querySelector('.play-list-length');
+const emptyMessageContainer = document.querySelector('.empty-message-container');
+const willStatus = document.querySelector('.will-status');
```

Comment:
초기에 이렇게 돔을 전체 선택 해버리면 동적으로 돔을 관리하기가 힘들지 않을까요?
```suggestion
const searchModalOpenButton = () => document.querySelector('#search-modal-open-button');
const modalClose = () => document.querySelector('.modal-close');
const modal = () => document.querySelector('.modal');
const youtubeSearchForm = () => document.querySelector('.youtube-search-form');
const youtubeSearchInput = () => document.querySelector('.youtube-search-input');
const modalVideoWrapper = () => document.querySelector('.modal .video-wrapper');
const mainVideoWrapper = () => document.querySelector('main > .video-wrapper');
const modalInner = () => document.querySelector('.modal-inner');
const recentSearchList = () => document.querySelector('.recent-search-list');
const playListLength = () => document.querySelector('.play-list-length');
const emptyMessageContainer = () => document.querySelector('.empty-message-container');
const willStatus = () => document.querySelector('.will-status');
```
이렇게 함수로 관리해주면 더 좋을 것 같아요!

Reply:
맞습니다 사실 중복을 줄이려고 시도해본 방법인데 말씀하신 이슈가 생길 수 있을것 같습니다!  생각하지 못한 부분을 캐치해주셔서 감사합니다!